### DISCUSSION TITLE
Installation using pip doesn't install the latest version

### DISCUSSION THREAD

**Discussion by zardadi** [pyproj4/pyproj#983] [General]
I installed pyproj using pip "pip install pyproj" and it doesn't install the latest version which is 3.2.1. 
I am installing on Windows and the installed version is :
Name: pyproj
Version: 3.0.1
Summary: Python interface to PROJ (cartographic projections and coordinate transformations library)
Home-page: https://github.com/pyproj4/pyproj

How can I install version 3.2.1?

**Comment by snowman2** [comment]
What Python version are you using? Related: #930 & #790

**Reply by zardadi** [reply]
Python 3.6.11

**Reply by snowman2** [reply]
Need 3.7+ to install the latest pyproj. With the next release, you will need 3.8+.

**Reply by zardadi** [reply]
You are right! it does the trick. Thanks
